### PR TITLE
Proof Integration Cases

### PR DESCRIPTION
**Pull request by Chirag-Khatri-CK**

### CODE REVIEW

File: src/components/Upload.vue [hunk 1]
```diff
@@ -1,62 +1,157 @@
 <template >
```
Comment (chayan007) [inline src/components/Upload.vue:1]: Based on file path, I cannot determine what `Upload` is this meant for ? Put it in proper folder !

File: src/components/Upload.vue [hunk 2]
```diff
@@ -1,62 +1,157 @@
...
 
 
 <script>
+
 export default {
   name: "Upload",
+  props:['files'],
+  data() {
+    return {
+      docfile: false,
```
Comment (chayan007) [inline src/components/Upload.vue:97]: Boolean variables should have name starting with `is_`

File: src/components/Upload.vue [hunk 3]
```diff
@@ -1,62 +1,157 @@
...
+      file: [],
```
Comment (chayan007) [inline src/components/Upload.vue:98]: If it's an array, the variable should be plural too.

File: src/components/Upload.vue [hunk 4]
```diff
@@ -1,62 +1,157 @@
...
+      filenames:null,
```
Comment (chayan007) [inline src/components/Upload.vue:99]: This variable is not used anywhere

File: src/components/case/CreateForm.vue [hunk 5]
```diff
@@ -182,7 +210,9 @@ export default {
       category: "",
       isAnonymous: false,
       // mention: "",
-      tagadded: false,
+      tagAdded: false,
+      files: [],
+      proofs: [],
```
Comment (chayan007) [inline src/components/case/CreateForm.vue:215]: This variable is not used anywhere

File: src/components/case/CreateForm.vue [hunk 6]
```diff
@@ -194,11 +224,20 @@ export default {
...
+      this.tagAdded = true;
     },
     removeTag(index) {
       this.tags.splice(index, 1);
-      this.tagadded = false;
+      this.tagAdded = false;
+    },
+
+    addFile(files) {
+      this.files.push(files);
```
Comment (chayan007) [inline src/components/case/CreateForm.vue:235]: Please remove `console.log()` apart from `.catch()` blocks. This is applicable for all such occurrences in this PR.

**Review by chayan007 — CHANGES_REQUESTED** [review]

File: src/components/Upload.vue (hunk 4 above)
Comment (Chirag-Khatri-CK) [reply src/components/Upload.vue:99]: Sir it is used in getFile() !

File: src/components/case/CreateForm.vue (hunk 5 above)
Comment (Chirag-Khatri-CK) [reply src/components/case/CreateForm.vue:215]: Sir, it is used for selecting categories!

File: src/components/Upload.vue (hunk 1 above)
Comment (Chirag-Khatri-CK) [reply src/components/Upload.vue:1]: Sir there is already upload.vue as a component in the case and may be used elsewhere !

File: src/components/Upload.vue (hunk 2 above)
Comment (Chirag-Khatri-CK) [reply src/components/Upload.vue:97]: Done Sir

File: src/components/Upload.vue (hunk 3 above)
Comment (Chirag-Khatri-CK) [reply src/components/Upload.vue:98]: files are passing in prop so I differentiate it!

File: src/components/Upload.vue (hunk 1 above)
Comment (chayan007) [reply src/components/Upload.vue:1]: `proof` folder

Comment (Chirag-Khatri-CK) [reply src/components/Upload.vue:1]: Done Sir

File: src/services/case.service.js [hunk 7]
```diff
@@ -1,99 +1,125 @@
...
+  let data = new FormData();
+  for (const [key, fileObj] of Object.entries(proofRequestBody))
+    data.append(key, fileObj);
+  return axios
+    .post(url, data, { headers: authHeaders })
+    .then((response) => {
+      return response.data;
+    })
+    .catch((error) => {
+      console.log("service level error: ", error.response);
```
Comment (chayan007) [inline src/services/case.service.js:106]: @Chirag-Khatri-CK throw error

**Pull request merged**